backstory/src/main/java/com/google/sps/perspective/PerspectiveManagerImpl.java

```diff
@@ -72,8 +72,9 @@ public PerspectiveManagerImpl(PerspectiveAPI perspectiveAPI) {
    * @throws NoAppropriateStoryException if no story from decision is considered appropriate
    */
   public PerspectiveDecision generateDecision(String story) throws NoAppropriateStoryException {
-    // TODO: Replace this fake behavior with a real API call, etc. Fake behavior here so teammates have stubbed
-    //       version of this class to use in integration. 
+    // TODO: Replace this fake behavior with a real API call, etc. Fake behavior here so teammates
+    // have stubbed
+    //       version of this class to use in integration.
```

Comment:
Make pretty updated this.

Reply:
Maybe move the string starting with "version" into the line above?

Reply:
Sure, updated.